src/Tippy.Util/EpochInfo.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+namespace Tippy.Util
+{
+    public class EpochInfo
```

Comment:
Let's move this to `Core` or `Rpc`. The idea is to make `Tippy.Util` pure utility package. It doesn't have to know anything about CKB blockchain or data structure.

Reply:
We can do that after merging.